rust/build.rs

```diff
@@ -71,6 +71,11 @@ impl Builder {
 }
 
 fn main() -> Result<(), Box<dyn std::error::Error>> {
+    // Download the protoc and set the path for tonic_build.
```

Comment:
Bump version to 1.0.1, and make another release.

Reply:
OK